### ISSUE TITLE
Unable to authenticate

### ISSUE THREAD

**Issue by gmyx** [dahlb/ha_blueair#100]
I';m trying to add my BlueAir 311i MAx, but getting invalid user. 

Logs:
2024-05-31 11:47:25.019 DEBUG (MainThread) [blueair_api.util_http] sending https://api.blueair.io/v2/user/[obfuscated@somewhere.com]/homehost/ requestheaders: {'X-API-KEY-TOKEN': '[obfuscated]'}
2024-05-31 11:47:25.176 DEBUG (MainThread) [blueair_api.util_http] response headers:<CIMultiDict('Content-Encoding': 'gzip', 'Content-Type': 'application/json;charset=UTF-8', 'Date': 'Fri, 31 May 2024 15:47:25 GMT', 'Server': 'nginx/1.12.1', 'Vary': 'Accept-Encoding,Accept-Encoding', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive')>
2024-05-31 11:47:25.176 DEBUG (MainThread) [blueair_api.util_http] response status: 200
2024-05-31 11:47:25.177 DEBUG (MainThread) [blueair_api.util_http] response raw: "api.blueair.io"
2024-05-31 11:47:25.177 DEBUG (MainThread) [blueair_api.util_http] sending https://api.blueair.io/v2/user/[obfuscated@somewhere.com]/login/ requestheaders: {'X-API-KEY-TOKEN': '[obfuscated]='}
2024-05-31 11:47:25.221 DEBUG (MainThread) [blueair_api.util_http] response headers:<CIMultiDict('Content-Encoding': 'gzip', 'Content-Type': 'application/json;charset=UTF-8', 'Date': 'Fri, 31 May 2024 15:47:25 GMT', 'Server': 'nginx/1.12.1', 'Vary': 'Accept-Encoding,Accept-Encoding', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive')>
2024-05-31 11:47:25.221 DEBUG (MainThread) [blueair_api.util_http] response status: 404
2024-05-31 11:47:25.222 DEBUG (MainThread) [blueair_api.util_http] response json: {'state': 404, 'message': 'This AuthUser does not exist.', 'requestedUri': '/v2/user/[obfuscated@somewhere.com]/login/', 'stack': '[obfuscated]', 'propagatedException': None}

**Comment by cromulus** [comment]
Same issue.

**Comment by nareddyt** [comment]
It was a problem with special characters in my password. I changed my password and it worked.

**Comment by epiech** [comment]
> It was a problem with special characters in my password. I changed my password and it worked.

Did you remove all special characters or did you limit them to certain ones?

**Comment by nareddyt** [comment]
Removed all

**Comment by cdmichaelb** [comment]
> It was a problem with special characters in my password. I changed my password and it worked.

this worked for me too

**Comment by SoccerCore11** [comment]
When trying to reset my password on blueair.com it is requiring special characters for complexity. How were you able to reset without them?

**Comment by cdmichaelb** [comment]
Use `!`

**Comment by gmyx** [comment]
I had a different problem. When I signed up, I used a different e-mail than my current. So my username is wrong. I went in my BlueAir profile, reset my e-mail to the old one, waited a bit and then put it back to my current. Now it works.

I guess the only thing that can be done is add to the documentation to ensure your password has only certain special characters and that your username matches profile e-mail.
